spec/acceptance/nginx_mail_spec.rb

```diff
@@ -5,15 +5,9 @@
 describe 'nginx::resource::mailhost define:' do
   has_recent_mail_module = true
 
-  if fact('os.family') == 'RedHat' && fact('os.release.major') == '8'
-    # EPEL had recent nginx-mod-mail package for CentOS 7 but not CentOS 8
-    # Stream.  The base packages use an older version of nginx that does not
-    # work with the acceptance test configuration.
-    has_recent_mail_module = false
-  end
+  has_recent_mail_module = false if fact('os.family') == 'RedHat' && fact('os.release.major') == '8'
```

Comment:
You can trim it down to a single statement
```suggestion
  has_recent_mail_module = fact('os.family') != 'RedHat' || fact('os.release.major') != '8'
```